Successfully updated:
Created Frame node fTtt2or2v
Created Frame node r0lv0CRZT
Created Text node AQ8qsCacp
Created Text node FQUxUWoUa
Created Frame node MZJ75fQkH
Updated attributes for node fTtt2or2v
Updated attributes for node r0lv0CRZT
Updated text for node AQ8qsCacp
Updated attributes for node AQ8qsCacp
Updated text for node FQUxUWoUa
Updated attributes for node FQUxUWoUa
Updated attributes for node MZJ75fQkH
Reordered node r0lv0CRZT within parent fTtt2or2v to index 0

XML Changes:
Index: node.xml
===================================================================
--- node.xml	Before
+++ node.xml	After
@@ -2564,10 +2564,63 @@
     nodeId="so_W61aYl"
     position="absolute"
     width="fit-content"
     height="fit-content"
     top="0px"
     left="0px"
     font="Inter"
 >
   something
 </Something>
+<Stack
+    nodeId="fTtt2or2v"
+    position="absolute"
+    width="400px"
+    height="300px"
+    top="0px"
+    left="0px"
+    <!-- background color string or project color style path (if starts with /) -->
+    backgroundColor="rgb(240, 240, 240)"
+    layout="stack"
+    gap="16px"
+    padding="20px"
+    stackDirection="vertical"
+    stackDistribution="center"
+    stackAlignment="center"
+>
+  <Frame
+      nodeId="r0lv0CRZT"
+      width="100%"
+      height="60px"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(100, 150, 200)"
+      borderRadius="8px"
+  >
+    <HeaderText
+        nodeId="AQ8qsCacp"
+        position="absolute"
+        width="fit-content"
+        height="fit-content"
+        top="0px"
+        left="0px"
+        font="Inter"
+    >
+      Header Text
+    </HeaderText>
+  </Frame>
+  <BodyContentGoesHere
+      nodeId="FQUxUWoUa"
+      width="fit-content"
+      height="fit-content"
+      font="Inter"
+  >
+    Body content goes here
+  </BodyContentGoesHere>
+  <Frame
+      nodeId="MZJ75fQkH"
+      width="100%"
+      height="1fr"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(255, 255, 255)"
+      borderRadius="4px"
+   />
+</Stack>
